docs/src/terminology.md

```diff
@@ -68,6 +68,14 @@ See [vote credit](terminology.md#vote-credit).
 
 A multicast network used to efficiently validate [entries](terminology.md#entry) and gain consensus.
 
+## delinquent
+
+When a validator does not submit a vote, it is known as delinquent. It is measured in the amount of blocks since a vote was last seen from that validator.
+
+## delinquent stake
+
+When a validator is considered delinquent, the amount of stake weight that has been delegated to it is added to the delinquent stake of the cluster.
```

Comment:
How about, "The amount of stake delegated to [delinquent](#delinquent) validators."